src/main/java/com/google/devtools/build/lib/rules/Alias.java

```diff
@@ -93,11 +93,6 @@ public Metadata getMetadata() {
   The <code>alias</code> rule creates another name a rule can be referred to as.
 </p>
 
-<p>
-  Aliasing only works for "regular" targets. In particular, <code>package_group</code>,
```

Comment:
Are you sure this works for `package_group` and `test_suite`? Is this functionality tested? `test_suite` rules are (unfortunately) special-cased in a number of ways e.g. in `bazel query` and other places expanding them (unfortunately, we have more than one) and I have no idea whether aliases are taken into account there.

Reply:
For `package_group` and `test_suite` I'm following the thought process described at https://github.com/bazelbuild/bazel/issues/6449#issuecomment-433712899.

Reply:
I think you are technically correct, but the fact that `package_group` is not a proper rule is a very obscure piece of information (except for us who have worked on Bazel since forever) and `test_suite` *is* a rule. Why not call this out explicitly?

(btw, `package_group` totally should be a rule. Totally my fault that it's not.)

Reply:
Call out what explicitly? Like, don't get into the details that `package_group` and `test_suite` are "special" but call them out.. how?

I don't mind adding some simple tests to check that alias works in a standard build use, but I wouldn't be enthused to check all the corner cases with query and who knows where else. :p

Reply:
Updated to just remove reference to `config_setting`, which is really the point of the bug. There's no evidence the comments on `package_group` and `test_suite` are confusing people, so let's just leave them be.